### ISSUE TITLE
Arm support

### ISSUE THREAD

**Issue by leiserfg** [opentracing-contrib/nginx-opentracing#391]
I build locally a docker image for Arm64 and it seems to work fine, can you please provide an official one?

**Comment by miry** [comment]
@leiserfg PR are welcome. I don't have a darwin/linux Arm64 machine to verify.

**Comment by miry** [comment]
I think there is existed support for multiple platforms and linux/arm64 included:

https://github.com/opentracing-contrib/nginx-opentracing/blob/master/.github/workflows/docker.yml#L14

Same stated in the Docker Hub:
https://hub.docker.com/layers/opentracing/nginx-opentracing/nginx-1.23.3/images/sha256-256c5d9331143f8bc9704967aa8d023a8018b1342f5b125841938bf56f0df4e3?context=explore

![image](https://user-images.githubusercontent.com/21104/218448396-3d212dc1-cd1c-4a32-b4e0-fc2789d8a771.png)

**Comment by leiserfg** [comment]
True, my fault, I forgot to mention that I was talking about the openresty image.

**Comment by leiserfg** [comment]
By the way, I also tried locally using a more recent version of ubuntu as the base image and it worked fine, that is very useful as now ubuntu (and debian) ships the datadog tracer in the repo so it works fine in amd64, and arm64.
